src/Orleans/Versions/Compatibility/ICompatibilityDirector.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+
+namespace Orleans.Versions.Compatibility
+{
+    public interface IVersionCompatibilityDirector
+    {
+        bool IsCompatible(ushort requestedVersion, ushort actualVersion);
```

Comment:
`actualVersion` is confusing, I had to look at the usage to understand what it is. Maybe `versionInUse` or something like that?